packages/core/src/context/MemoryStore.example.ts

```
@@ -0,0 +1,352 @@
+    case 'add':
+      await addMemoryExample();
+      break;
+    case 'integration':
+      await integrationExample();
+      break;
+    case 'multiple':
+      await multipleFilesExample();
+      break;
+    case 'all':
+      await basicExample();
+      console.log('\n' + '='.repeat(50) + '\n');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)